recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -44,11 +44,8 @@ def requirements(self):
 
     def validate(self):
         if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
-
-        if tools.is_apple_os(self.settings.os):
-            # FIXME: add Apple support
-            raise ConanInvalidConfiguration("This recipe does not support Apple.")
+            # Set as required in crashpad CMake file
+            tools.check_min_cppstd(self, 14)
```

Comment:
Maybe be less restrictive for the rest?

```suggestion
            tools.check_min_cppstd(self, 14 if tools.is_apple_os(self.settings.os) else 11)
```

Reply:
Thanks for the tip and link! I'll change, because it actually looks more verbose.

Actually this is not apple specific change. I decided to follow specification in project's CMake file:
https://github.com/getsentry/crashpad/blob/71bcaad4cf30294b8de1bfa02064ab629437163b/CMakeLists.txt#L67